pluginProjects/FileLoaders/SGM2010FastFileLoader/SGM2010FastFileLoaderPlugin.cpp

```diff
@@ -139,14 +139,12 @@ bool SGM2010FastFileLoaderPlugin::load(AMScan *scan, const QString &userDataFold
 	QString scalerFile = "";
 	QFileInfo scalerFileInfo;
 	if(colNames1.contains("scaler_fileOffset")){
-		foreach(QString afp, scan->additionalFilePaths())
```

Comment:
@XuetaoLiu-CLS, not all of these are identically the same.  Are you sure that the result will be preserved by these changes?